### PR TITLE
new icon: AArch64 (original, plain)

### PR DESCRIPTION
**Pull request by Panquesito7**
I added an icon for [AArch64](https://en.wikipedia.org/wiki/AArch64) (also see https://github.com/TheAlgorithms/AArch64_Assembly).
I didn't find out any official logos, so I picked up the logo and converted it to `.svg` format from [this website](https://www.konsulko.com/home-servers-aarch64-and-you-well-me/) and slightly modified it. Resized using the [`resize-image`](https://www.iloveimg.com/resize-image/resize-svg) tool. 🙂

This is how they look:

AArch64 original icon AArch64 plain icon

They look kinda stretched, but not sure what to do.
If I missed up anything, please let me know. Thank you.

### CODE REVIEW

**Comment by amacado** [comment]
That's a tough decision. I'm not sure if `AArch64` fulfills our guidelines about which type of icons should be added to this project (see https://github.com/devicons/devicon/discussions/353). If I got it right it's neither a language, nor a development tool or library so I'm not sure if it fit's in the purpose of this repository. On the other hand it clearly has something in common with software development. @Thomas-Boi what do you think? Open discussion for all users of devicon. Share your opinion! :)

**Comment by maltejur** [comment]
> That's a tough decision. I'm not sure if `AArch64` fulfills our guidelines about which type of icons should be added to this project (see #353). If I got it right it's neither a language, nor a development tool or library so I'm not sure if it fit's in the purpose of this repository. On the other hand it clearly has something in common with software development. @Thomas-Boi what do you think? Open discussion for all users of devicon. Share your opinion! :)

Well even if it isn't a full fletched "programming language" in that sense, I still think if there even are icons for IDE's, this should definitely also have a place here.

**Comment by Panquesito7** [comment]
It's somewhat like a hardware-thing but a programming language too, but as @maltejur said

> Well even if it isn't a full-fletched "programming language" in that sense, **I still think if there even are icons for IDE's**, this should definitely also have a place here.

I actually think this should be added here. As you can see in the [AArch64 Assembly](https://github.com/TheAlgorithms/AArch64_Assembly) repository in [The Algorithms](https://github.com/TheAlgorithms), it **is** a programming language. However, Wikipedia states it's somewhat hardware-based (the image says it all). You can search [through GitHub](https://github.com/search?q=language%3A+assembly&ref=simplesearch) and you'll see this is a programming language. Not sure if it should be AArch64 or Assembly in this case (or AArch64 Assembly, which is both hardware and software). _Offtopic:_ The use-case of this PR/icon is for a new website we're building, and we don't have icons for this programming language.

TL;DR: I think we should add this to the repository. As @maltejur said, there are icons for IDE's, so this icon should fit well here, too.

**Comment by Thomas-Boi** [comment]
Hi @amacado,

> As you can see in the AArch64 Assembly repository in The Algorithms, it is a programming language. However, Wikipedia states it's somewhat hardware-based (the image says it all). You can search through GitHub and you'll see this is a programming language

I think this is true. AArch64 does look like it's some form of an assembly language so I have no issue with accepting this PR (after checking the icons of course)

**Review by Thomas-Boi — APPROVED** [review]
I am fine with this request. AArch64 does seem like a programming language and the files are created properly.

**Comment by Panquesito7** [comment]
_Offtopic: @amacado I really love this project, it's pretty cool. I'd like to become a member of Devicons. I'll help with reviewing pull requests, adding new icons (in a pull request), etc.. Waiting on your response. Thank you!_ 🙂

**Comment by amacado** [comment]
@Panquesito7 you're welcome! I will contact you via mail so we can discuss some details 👍🏻
